pkg/repositories/gormimpl/artifact.go

```diff
@@ -53,9 +53,12 @@ func (h *artifactRepo) Get(ctx context.Context, in models.ArtifactKey) (models.A
 	defer timer.Stop()
 
 	var artifact models.Artifact
-	result := h.db.Preload("ArtifactData").Preload("Partitions").First(&artifact, &models.Artifact{
-		ArtifactKey: in,
-	})
+	result := h.db.Preload("ArtifactData").Preload("Partitions").Preload("Tags").
```

Comment:
just to double check because this bit us in admin in the past - these fields are all indexed?

Reply:
yes confirmed with a `explain analyze` manual sql statement.
```
 Nested Loop  (cost=1.11..244.37 rows=1 width=37) (actual time=0.476..2.751 rows=2 loops=1)
   ->  Index Only Scan using tags_pkey on tags  (cost=0.56..72.84 rows=20 width=37) (actual time=0.030..0.294 rows=240 loops=1)
         Index Cond: (tag_name = 'LATEST'::text)
         Heap Fetches: 240
   ->  Index Scan using artifacts_pkey on artifacts  (cost=0.56..8.57 rows=1 width=37) (actual time=0.010..0.010 rows=0 loops=240)
         Index Cond: (artifact_id = tags.artifact_id)
         Filter: ((deleted_at IS NULL) AND (dataset_uuid = '7446bc0f-3615-4ae4-9d4d-8f04389e13a3'::uuid))
         Rows Removed by Filter: 1
```